tests/integration/test_parameters_integration.py

```diff
@@ -91,7 +91,9 @@ def func2(x, y):
 
     return eng, prog, params
 
-
+@pytest.mark.backends("fock", "tf","gaussian")
+#TODO: make Catstate work with bosonic
+#Currently encounters a problem trying to apply np.cos when phi is a MeasuremedParameter
```

Comment:
Can you elaborate what is going wrong here?

Reply:
In line 126 (new numbering) the Catstate operation takes the MeasuredParameter r[0],r[1] as alpha and phi values. In the Catstate preparation, we take np.cos(phi) and it seems like that function can't be applied to MeasuredParameter

Reply:
Can we just make so that they have the same signature?

Reply:
Having discussed, it seems to be an issue of using np.cos vs pf.cos in the catstate preparation method. We will bypass the test for now.